AK/EnumBits.h

```diff
@@ -20,62 +20,62 @@
+            static_cast<Type>(lhs) ^ static_cast<Type>(rhs));              \
+    }                                                                      \
+                                                                           \
+    [[nodiscard]] Prefix constexpr Enum operator~(Enum rhs)                \
+    {                                                                      \
+        using Type = UnderlyingType<Enum>;                                 \
+        return static_cast<Enum>(                                          \
+            ~static_cast<Type>(rhs));                                      \
+    }                                                                      \
+                                                                           \
+    Prefix constexpr Enum& operator|=(Enum& lhs, Enum rhs)                 \
+    {                                                                      \
+        using Type = UnderlyingType<Enum>;                                 \
+        lhs = static_cast<Enum>(                                           \
+            static_cast<Type>(lhs) | static_cast<Type>(rhs));              \
+        return lhs;                                                        \
+    }                                                                      \
+                                                                           \
+    Prefix constexpr Enum& operator&=(Enum& lhs, Enum rhs)                 \
+    {                                                                      \
+        using Type = UnderlyingType<Enum>;                                 \
+        lhs = static_cast<Enum>(                                           \
+            static_cast<Type>(lhs) & static_cast<Type>(rhs));              \
+        return lhs;                                                        \
+    }                                                                      \
+                                                                           \
+    Prefix constexpr Enum& operator^=(Enum& lhs, Enum rhs)                 \
+    {                                                                      \
+        using Type = UnderlyingType<Enum>;                                 \
+        lhs = static_cast<Enum>(                                           \
+            static_cast<Type>(lhs) ^ static_cast<Type>(rhs));              \
+        return lhs;                                                        \
+    }                                                                      \
+                                                                           \
+    Prefix constexpr bool has_flag(Enum value, Enum mask)                  \
+    {                                                                      \
+        using Type = UnderlyingType<Enum>;                                 \
+        return static_cast<Type>(value & mask) != static_cast<Type>(mask); \
```

Comment:
As discussed on discord, this should be: 

```suggestion
        return static_cast<Type>(value & mask) == static_cast<Type>(mask); \
```